tests/config/test_data.config

```diff
@@ -249,11 +249,17 @@ params {
             }
         }
         'generic' {
+            'csv' {
+                test_csv = "${test_data_dir}/generic/csv/test.csv"
```

Comment:
We should probably flatten these keys at some point because it doesn't make sense to have a `csv` key for one or two files.

Reply:
Makes sense to me. Should I change it right away?